inc/templates/post-list-single-2-cols.php

```diff
@@ -41,12 +41,13 @@
 		<?php endif; ?>
 
 		<?php if ($show_meta): ?>
-			<?php echo_post_meta($post,array('date','sources'),$order); ?>
+			<?php echo_post_meta($post,array('date','sources','show_summary_translated_by_odc_team'),$order); ?>
 		<?php endif; ?>
 
-		<?php if ($show_summary_translated_by_odc_team): ?>
+		<!-- <?php if ($show_summary_translated_by_odc_team): ?>
```

Comment:
@S-mardii please do not leave commented code behind. If we do not need this anymore then it should be removed.